tests/login/file.txt

```diff
@@ -0,0 +1 @@
+Hello TMS!!!
```

Comment:
Please create a separate folder for test data/files e.g. "resources" or "test-data" and keep .png and .txt files there